docs/releases/3.0.txt

```diff
@@ -76,6 +76,10 @@ Minor features
   to mirror the existing
   :meth:`~django.contrib.auth.models.User.get_group_permissions()` method.
 
+* Added HTML ``autocomplete`` attribute to widgets of username, email and
```

Comment:
serial comma? :wink: `username, email, and password`

Reply:
Done!